README.md

```diff
@@ -1,249 +1,98 @@
-[![Build Status](https://travis-ci.com/carpentries/workshop-template.svg?branch=gh-pages)](https://travis-ci.com/carpentries/workshop-template)
+[![Build Jekyll site](https://github.com/esciencecenter-digital-skills/next-workshop-template/actions/workflows/jekyll.yml/badge.svg)](https://github.com/esciencecenter-digital-skills/next-workshop-template/actions/workflows/jekyll.yml)
 
-# The Carpentries Workshop Template
+# The Workshop Template
 
-This repository is The Carpentries' ([Software Carpentry][swc-site], [Data Carpentry][dc-site], and
-[Library Carpentry][lc-site]'s) template for creating websites for workshops.
+This repository is a template for creating a workshop page for [eScience Center
+Digital Skills][ds-site], and the Carpentries [Software Carpentry][swc-site],
+[Data Carpentry][dc-site], [Library Carpentry][lc-site].
+We are committed to offering a pleasant setup experience for our learners and
+organizers. If you run into problems, or would like to suggest improvements,
+please [submit an issue][issues] or [mail us][email].
 
-1. **Please _do not fork this repository directly on GitHub._** Instead, please use GitHub's
-   "template" function following [the instructions below](#creating-a-repository) to copy this
-   `workshop-template` repository and customize it for your workshop.
+## Use this template
 
-2. Please *do your work in your repository's `gh-pages` branch*, since that is what is
-   [automatically published as a website by GitHub][github-project-pages].
+1. **Please _do not fork this repository directly on GitHub._** Instead,
+   use GitHub's function to copy this template repository and customize it for
+   your workshop. On this page
+   (<https://github.com/esciencecenter-digital-skills/next-workshop-template>),
+   click on the green `Use this template` button (top right).
 
-3. Once you are done, please also [let us know][email] the workshop URL. If this is a self-organised
-   workshop, you should also [fill in the self-organized workshop
-   form][self-organized-workshop-form] (if you have not already done so), so we can keep track of
-   all workshops. We build the list of workshops on our websites from the data included in your
-   `index.md` page. We can only do that if you [customize][customization] that page correctly *and*
-   let us know the workshop URL.
+2.  Select the owner for your new repository. This will probably be you, but may
```

Comment:
```suggestion
2.  Select the owner for your new repository. This should be
```
```suggestion
2.  Select the owner for your new repository. This will probably be you, but may
```